bridge-deploy/tests/test_cli.py

```diff
@@ -0,0 +1,48 @@
+import pytest
+
+from click.testing import CliRunner
+
+from bridge_deploy.cli import main
+
+
+@pytest.fixture
+def runner():
+    return CliRunner()
+
+
+def test_deploy_home(runner):
+
+    # TODO: Add actual working addresses and contracts
+    result = runner.invoke(
+        main,
+        args=(
+            "deploy-home"
+            " --jsonrpc test "
+            " --validator-set-address 0x0000000000000000000000000000000000000000"
+            " --block-reward-address 0x0000000000000000000000000000000000000000"
+            " --owner-address 0x0000000000000000000000000000000000000000"
```

Comment:
Shouldn't this fail with owner = addres(0) ?
https://github.com/poanetwork/poa-bridge-contracts/blob/master/contracts/upgradeable_contracts/erc20_to_native/HomeBridgeErcToNative.sol#L142

Reply:
Yes. It does.
We still need to hook this up to the validator set proxy and block reward contract.
These need to be merged / deployed first.
Are we going to deploy the reward contract via the chainspec?

Reply:
Changed to address(2) for now, but the test still fails. Maybe a builtin is not considered a valid contract. I'll investigate.

Reply:
I am not sure how we will deploy the reward contract. I think a regular deployment is the easiest

Reply:
Reward Contract will be deployed as part of the bridge deployer now.